addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -1948,6 +2193,44 @@ const actions = {
             });
         }
     },
+    /**
+     * Fetch attachments linked to a record. Useful for populating the store
+     * with these attachments, which are used by attachment box in the chatter.
+     *
+     * @param {Object} param0
+     * @param {function} param0.dispatch
+     * @param {Object} param0.env
+     * @param {Object} param0.state
+     * @param {string} threadLocalId
+     */
+    async _fetchThreadAttachments(
```

Comment:
wait... that's the exact same code as `fetchThreadAttachments`!

Reply:
True, wtf Oo